saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/formSection.js

```diff
@@ -0,0 +1,197 @@
+/*
+ * Form section
+ * 
+ * @author Łukasz Pawełczak
+ * 
+ * 
+ * @param userOptions - object with structure
+ * 
+ *  {
+ *      formSectionId - id of form section 
+ *      infoSectionId - id of info section
+ *  }
+ * 
+ */
+function FormSection(userOptions) {
+   
+    
+    
+    var defaultOptions = {
+        formSectionId: "",
+        infoSectionId: ""
+    };
+    
+    var options = userOptions || defaultOptions;
+    
+    init();
+    
+    
+    //------------------------ FUNCTIONS --------------------------
+    
+    function init() {
+        showPublishedValues();
+        initOpenFormSection();
+        initCloseFormSection();
+    }
+    
+    
+    /* 
+     * Assigns buttons to open form section on event click.
+     */ 
+     function initOpenFormSection() {
+    
+        var $formSection = $(options.formSectionId);
+        
+        $(options.infoSectionId).click(function(event) {    
+            
+            event.preventDefault();
+            
+            $formSection.slideDown(400, function() {});
+        });
+        
+    }
+     
+     /* 
+      * Assigns buttons to close form section on event click.
+      */ 
+     function initCloseFormSection() {
+         
+         var $formSection = $(options.formSectionId),
+             $setSection = $(options.infoSectionId);
+         
+         $(document).mouseup(function(e) {
+             
+             if (!$formSection.is(e.target) && !$setSection.is(e.target) // if the target of the click isn't the container...
+                     && $formSection.has(e.target).length === 0 && $setSection.has(e.target).length === 0) // ... nor a descendant of the container
+             {
+                 closeFormSection();
+             }
+             
+         });
+     }
+     
+     /*
+      * Closes form section
+      */
+     function closeFormSection() {
+         var $formSection = $(options.formSectionId);
+
+         $formSection.find(".form-group").each(function() {
```

Comment:
I completely don't understand this section... this removing of input field values...